test/data/ReqMgr/validate-test-wfs.py

```diff
@@ -240,6 +242,62 @@ def compareSpecial(d1, d2, key=None):
     return 'ok'
 
 
+def twClosure(replaceWhiteSpace=False,
+              breakLongWords=False,
+              width=120,
+              initialIndent=''):
+    """
+    Deals with indentation of dictionaries with very long key, value pairs.
+    params: look at TextWrapper documentation
```

Comment:
I'm sure you can give a very short description for all these 4 parameters

Reply:
Parameter description was added to the doc string with the latest commit.